doc/getting-started/agents_utils/agents_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+LOGS_DIR=/home/indy/logs
+
+mkdir -p $LOGS_DIR
+
+startagent () {
+    local NAME=$1
+    local PORT=$2
+    echo Starting agent \"$NAME\" on port $PORT
+    python3 "/usr/local/lib/python3.5/dist-packages/indy_client/test/agent/$NAME.py" --port $PORT > $LOG_DIR/$NAME.log
```

Comment:
@stevengubler @ashcherbakov is it script for running old, python-based Agent for getting started?